platform/ios/src/MGLMapView.mm

```diff
@@ -1070,6 +1070,8 @@ - (void)sleepGL:(__unused NSNotification *)notification
     {
         self.dormant = YES;
 
+        [MGLOfflineStorage sharedOfflineStorage].mbglFileSource->pause();
```

Comment:
Just a reminder to move this code into MGLOfflineStorage itself if possible: https://github.com/mapbox/mapbox-gl-native/pull/8125#discussion_r102102611.